blazor/datagrid/excel-like-filter.md

```diff
@@ -241,4 +241,57 @@ Before opening a filter dialog for each column, the `OnActionBegin` event will b
 
 N> [View Sample in GitHub.](https://github.com/SyncfusionExamples/blazor-datagrid-customize-filter-popup-using-css)
 
+## Customize the CheckboxListData in filter Popup
+ 
+To pass custom datasource for the checkbox list data in a Grid column filter you can use [CheckboxListData](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.ActionEventArgs-1.html#Syncfusion_Blazor_Grids_ActionEventArgs_1_CheckboxListData) inside [OnActionBegin](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridEvents-1.html#Syncfusion_Blazor_Grids_GridEvents_1_OnActionBegin) event of the Grid.
```

Comment:
Mention as Using FilterItemTemplate feature, custom values can be displayed in Checkbox/excel filter dialog. But default searching will perform based on the column's field value only. So to perform search operation, we have provided support to CheckBoxListData where custom data list can be passed to filter dialog and displayed in it